src/implementations/twig/components/page-header-core/page-header-core.story.js

```diff
@@ -13,98 +11,106 @@ import dataBreadcrumbLong from '@ecl/specs-component-breadcrumb/demo/data';
 import notes from './README.md';
 
-const preparePageHeaderCore = (data, desc, meta, img) => {
-  data.breadcrumb = dataBreadcrumbLong;
-  data.title = text('title', data.title, tabLabels.required);
-  if (meta) {
-    data.meta = text('meta', data.meta, tabLabels.optional);
+const getArgTypes = (data) => {
+  const argTypes = {};
+  argTypes.title = {
+    name: 'title',
+    type: { name: 'string', required: true },
+    defaultValue: data.title,
+    description: 'The page title',
+    table: {
+      type: { summary: 'string' },
+      defaultValue: { summary: '' },
+      category: 'Content',
+    },
+  };
+  if (data.description) {
+    argTypes.description = {
+      name: 'description',
+      type: 'string',
+      defaultValue: data.description,
+      description: 'The page introduction',
+      table: {
+        type: { summary: 'string' },
+        defaultValue: { summary: '' },
+        category: 'Content',
+      },
+    };
   }
-  if (desc) {
-    data.description = text(
-      'description',
-      data.description,
-      tabLabels.optional
-    );
+  if (data.meta) {
+    argTypes.meta = {
+      name: 'meta',
+      type: 'string',
+      defaultValue: data.meta,
+      description: 'The page metadata',
+      table: {
+        type: { summary: 'string' },
+        defaultValue: { summary: '' },
+        category: 'Content',
+      },
+    };
   }
-  if (img) {
-    data.background_image = boolean(
-      'background_image',
-      data.background_image,
-      tabLabels.required
-    );
-    data.background_image_url = text(
-      'background_image_url',
-      data.background_image_url,
-      tabLabels.required
-    );
+  if (data.background_image_url) {
+    argTypes.background_image_url = {
+      name: 'background image',
+      type: 'string',
+      defaultValue: data.background_image_url,
+      description: 'The background image url',
+      table: {
+        type: { summary: 'string' },
+        defaultValue: { summary: '' },
+        category: 'Content',
+      },
+    };
   }
-  data.breadcrumb.icon_file_path = optionsKnob(
-    'breadcrumb.icon_file_path',
-    { current: defaultSprite, 'no path': '' },
-    defaultSprite,
-    { display: 'inline-radio' },
-    tabLabels.required
-  );
-  data.breadcrumb.ellipsis_label = text(
-    'breadcrumb.ellipsis_label',
-    data.breadcrumb.ellipsis_label,
-    tabLabels.required
-  );
-  data.breadcrumb.navigation_text = text(
-    'breadcrumb.navigation_text',
-    data.breadcrumb.navigation_text,
-    tabLabels.required
-  );
-  data.breadcrumb.links.forEach((item, i) => {
-    item.label = text(
-      `data.breadcrumb.links[${i}].label`,
-      item.label,
-      tabLabels.required
-    );
-    item.path = text(
-      `data.breadcrumb.links[${i}].path`,
-      item.path,
-      tabLabels.required
-    );
-  });
-  getExtraKnobs(data);
 
-  return data;
+  return argTypes;
+};
+
+const prepareData = (data, args) => {
+  data.breadcrumb = dataBreadcrumbLong;
```

Comment:
I see that we did not do when we worked on the breadcrumbs, but we have now the EU storybook and in theory we have, or better had, different specs for EU (European union instead of European commission, basically), not a big deal but it looks like we have the time and the chance to do this now. 
Up to you to do this in the context of this pr, otherwise we can create a separate ticket for it, or we can simply ignore this.. :)